### PR TITLE
Flask deployment

### PR DESCRIPTION
**Pull request by justmesam**

### CODE REVIEW

**Comment by rnjane** [comment]
Nice atomic commits, but try at all times to make meaningful commits.
Test coverage is good, but you can improve it.
Nice code! well structured

**Comment by Gidraff** [comment]
Awesome commit messages.Work to fix all the changes

File: tests/test_flask.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        """logout helper   """
+        return self.app.get('/logout',
+                            follow_redirects=True)
+
+    def test_misc_pages(self):
+        """tests the response of the miscellaneous pages"""
+        ## >> index page << ##
+        result = self.app.get('/', follow_redirects=True)
+        result1 = self.app.get('/index', follow_redirects=True)
+        self.assertEqual(result.status_code, 200)
```
Comment (clemwek) [inline tests/test_flask.py:39]: You can also add test for the returned data, it may be loading a wrong page.

**Review by clemwek — COMMENTED** [review]
Great work :)

**Pull request merged**